Updating incorrect information in the glossary (#8043)
istio/istio.io: content/en/docs/reference/glossary/trust-domain.md

test: n/a
---

[Trust domain](https://spiffe.io/spiffe/concepts/#trust-domain) corresponds to the trust root of a system and is part of a workload identity.

Istio uses a trust domain to create all [identities](/docs/reference/glossary/#identity) within a mesh.

> 

Reviewer: ```suggestion
```

For example in `spiffe://mytrustdomain.com/ns/default/sa/myname` the substring `mytrustdomain.com` specifies that the workload is from a trust domain called `mytrustdomain.com`.

> Depending upon your configuration, you can choose to have a single trust domain for the entire mesh across clusters or a trust domain for each cluster, as long as the clusters share the same root of trust.

Reviewer 2: This seems overly specified, since you could also have several groups of clusters each using the same trust domain.

Maybe we can just say you can have one or more trust domains in a single mesh.

Reviewer: How about:
```suggestion
You can have one or more trust domains in a multicluster mesh, as long as the clusters share the same root of trust.
```
